guides/common/Makefile

```diff
@@ -27,7 +27,7 @@ clean:
 	@rm -rf "$(DEST_DIR)" "$(DEST_PDF)" ../common/build.adoc
 
 browser: html
-	$(BROWSER) "file://$(realpath $(ROOTDIR)/$(DEST_HTML))"
+	xdg-open "file://$(realpath $(ROOTDIR)/$(DEST_HTML))"
```

Comment:
@melcorr if you did "make browser" it was probably not opening Firefox for you, not sure. It did not work for @rahulbajaj0509 and this fixes it.

Reply:
@lzap I just pointed firefox command at the build path - I didn't do make browser!